R/dot-net-wrapper.R

```diff
@@ -1,12 +1,16 @@
-#' @title DotNetRwapper
+#' @title Wrapper class for `.NET` objects
 #' @docType class
-#' @description  Wrapper class for .net Object
 #'
-#' @section Methods:
-#' \describe{
-#'   \item{wrapProperty}{Simple way to wrap a get;set; .NET property}
-#'   \item{wrapReadOnlyProperty}{Simple way to wrap a get; .NET Read-Only property}
-#'   }
```

Comment:
These are private methods, so no need to document them at the top level. 

Moved them to comments for respective method definitions.